packages/discord.js/src/structures/CategoryChannel.js

```diff
@@ -8,9 +8,22 @@ const CategoryChannelChildManager = require('../managers/CategoryChannelChildMan
  * @extends {GuildChannel}
  */
 class CategoryChannel extends GuildChannel {
+  /**
+   * The id of the parent of this channel.
```

Comment:
```suggestion
   * The id of the parent of this category.
```

as it's already custom doc

Reply:
Invalid as category channels cannot have parents